css-two-column-layout/index.html

```diff
@@ -0,0 +1,79 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+  <meta charset="UTF-8">
+  <meta name="viewport" content="width=device-width, initial-scale=1.0">
+  <meta http-equiv="X-UA-Compatible" content="ie=edge">
+  <title>CSS Challenge - Two Column Layout</title>
+  <link rel="stylesheet" href="fonts.css">
+  <link rel="stylesheet" href="reset.css">
+  <link rel="stylesheet" href="styles.css">
+</head>
+<body>
+  <div class="container">
+    <div class="head">
+      <div class="title-head">
+        <h1>SCALE, SPEED, AGILITY</h1>
+      </div>
+      <div class="p-head">
+        <p>
+          We've dedicated thousands of man hours to making healthcare
+          institutions in the Helium network run 10x more effeciently and deliver
+          an unparalleled quality of care.
+        </p>
+      </div>
+    </div>
+    <div class="row">
+      <div class="column-fourth">
```

Comment:
because each column is taking up half the row, `column-half` would be a better semantic name.